pyo3-ffi/src/cpython/ceval.rs

```diff
@@ -2,10 +2,22 @@ use crate::cpython::pystate::Py_tracefunc;
 use crate::object::{freefunc, PyObject};
 use std::os::raw::c_int;
 
+#[cfg(any(PyPy, Py_3_11))]
+opaque_struct!(_PyInterpreterFrame);
+
 extern "C" {
     // skipped non-limited _PyEval_CallTracing
 
+    #[cfg(not(Py_3_11))]
     pub fn _PyEval_EvalFrameDefault(arg1: *mut crate::PyFrameObject, exc: c_int) -> *mut PyObject;
+
+    #[cfg(Py_3_11)]
+    pub fn _PyEval_EvalFrameDefault(
+        tstate: *mut crate::PyThreadState,
+        frame: *mut crate::_PyInterpreterFrame,
+        throwval: c_int,
```

Comment:
It looks like the header gives this argument a different name?

```suggestion
        exc: c_int,
```